agent/functional_tests/tests/functionaltests_unix_test.go

```diff
@@ -113,12 +113,15 @@ func TestTaskCleanupDoesNotDeadlock(t *testing.T) {
 
 	// Run two Tasks after cleanup, as the deadlock does not consistently occur after
 	// after just one task cleanup cycle.
-	for i := 0; i < 3; i++ {
+	for i := 0; i < 2; i++ {
```

Comment:
To match the comment and the equivalent windows test. However, we can switch to three tasks if we'd rather.